CONTRIBUTING.md

```diff
@@ -0,0 +1,87 @@
+# Community Guidelines
```

Comment:
Hey @exegeteio - this appears to be the same content as the Code Of Conduct file ;)

Reply:
I knew it was too easy.  I copied over top of what I wrote on accident.  Should be updated.  Good catch.  👍